Gems/Atom/Feature/Common/Assets/Materials/Types/NewPBR/NewPBR_ForwardPass.azsl

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+
+#define ENABLE_CLEAR_COAT       0
+#define ENABLE_TRANSMISSION     0
+#define FORCE_OPAQUE            1
+
+// Options
+#include <Atom/Features/ShaderQualityOptions.azsli>
+#include <Atom/Features/PBR/LightingOptions.azsli>
+
+#include "NewPBR_Common.azsli"
+#include <Atom/Features/PBR/DefaultObjectSrg.azsli>
+
+// ---------- Material Parameters ----------
+
+COMMON_OPTIONS_BASE_COLOR()
+COMMON_OPTIONS_ROUGHNESS()
+COMMON_OPTIONS_METALLIC()
+COMMON_OPTIONS_SPECULAR_F0()
+COMMON_OPTIONS_NORMAL()
+COMMON_OPTIONS_OCCLUSION()
+COMMON_OPTIONS_EMISSIVE()
+
+
+// Pass SRG
+#include <Atom/Features/PBR/ForwardPassSrg.azsli>
+
+// Pass Output
+#include <Atom/Features/PBR/ForwardPassOutput.azsli>
+
+// Utility
+#include <Atom/Features/ColorManagement/TransformColor.azsli>
+
+// Shader Stitching
+#include "NewPBR_Vertex.azsli"
+#include "NewPBR_PixelGeometry.azsli"
+#include "NewPBR_SurfaceData.azsli"
+#include "NewPBR_Surface.azsli"
+#include "NewPBR_LightingData.azsli"
```

Comment:
Last time I looked, the LightingData struct was common across all the lighting models. Are you are changing something about the struct that needs one specific to NewPBR?

Reply:
If not, then we should change NewPBR_LightingData.azsli and NewPBR_Lighting.azsli to be common files.